pkg/regressionallowances/regressions_4.18.json

```
@@ -1 +1,428 @@
-[]
+[
+  {
+    "JiraComponent": "OLM",
+    "TestID": "openshift-tests:0c0cc3c8ed00082e1c8c423d61129c21",
+    "TestName": "[sig-olmv1][OCPFeatureGate:NewOLM][Skipped:Disconnected] OLMv1 operator installation should fail to install a non-existing cluster extension [Suite:openshift/conformance/parallel]",
+    "JiraBug": "https://issues.redhat.com/browse/OCPBUGS-49860",
+    "ReasonToAllowInsteadOfFix": "The OLMv1 subsystem is new to OCP 4.18. Operator-controller is encountering flakiness when connecting to catalogd securely. The catalogd TLS service certificate in unable to be verified by the service-ca. Manifest changes in certificate loading and extra logs were added to determine the cause of the errors.",
```

ITYM
```suggestion
    "ReasonToAllowInsteadOfFix": "The OLMv1 subsystem is new to OCP 4.18. Operator-controller is encountering flakiness when connecting to catalogd securely. The catalogd TLS service certificate is unable to be verified by the service-ca. Manifest changes in certificate loading and extra logs were added to determine the cause of the errors.",
```

and in every copy below i guess...

---

global search/replace...